client/src/pages/SignUp/useStyles.ts

```diff
@@ -6,20 +6,29 @@ const useStyles = makeStyles(() => ({
     '& .MuiInput-underline:before': {
       borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
     },
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#FAFAFA',
+  },
+  container: {
+    height: '100%',
+    background: '#fff',
+    borderRadius: 10,
+    boxShadow: '1px -1px 23px 2px rgba(218,225,240,0.75)',
+    webkitboxShadow: '1px -1px 23px 2px rgba(218,225,240,0.75)',
+    mozboxShadow: '1px -1px 23px 2px rgba(218,225,240,0.75)',
   },
   authWrapper: {
-    display: 'flex',
-    alignItems: 'flex-start',
-    justifyContent: 'space-between',
-    flexDirection: 'column',
-    minHeight: '100vh',
     paddingTop: 23,
+    display: 'flex',
```

Comment:
```suggestion
```
Flex display has been already added to the `Box` component this CSS class has been applied to